# HG changeset patch
# User John W. Eaton <jwe@octave.org>
# Date 1486074359 18000
#      Thu Feb 02 17:25:59 2017 -0500
# Node ID e66b3bfea38045dcfc7f9571f7fb6ccfe9ee1609
# Parent  334119c390b37e6fa4c547c9a6ab75bf93dbf058
# Parent  900ae902aa9007d528cf7d47ae8821a449ef1c80
maint: Merge stable to default.

diff --git a/NEWS b/NEWS
--- a/NEWS
+++ b/NEWS
@@ -258,17 +258,17 @@ Summary of important user-visible change
     be removed from Octave 4.6 (or whatever version is the second major
     release after 4.2):
 
       Function             | Replacement
       ---------------------|------------------
       bitmax               | flintmax
       mahalanobis          | mahal in Octave-Forge statistics pkg
       md5sum               | hash
-      octve_config_info    | __octave_config_info__
+      octave_config_info   | __octave_config_info__
       onenormest           | normest1
       sleep                | pause
       usleep               | pause
       wavread              | audioread
       wavwrite             | audiowrite
 
  ** The following functions were deprecated in Octave 3.8 and have been
     removed from Octave 4.2.
diff --git a/scripts/plot/util/print.m b/scripts/plot/util/print.m
--- a/scripts/plot/util/print.m
+++ b/scripts/plot/util/print.m
@@ -310,27 +310,27 @@
 ## @seealso{saveas, hgsave, orient, figure}
 ## @end deftypefn
 
 function print (varargin)
 
   opts = __print_parse_opts__ (varargin{:});
 
   folder = fileparts (opts.name);
-  if (! isempty (folder) && ! exist (folder,"dir"))
+  if (! isempty (folder) && ! exist (folder, "dir"))
     error ("print: directory %s does not exist", folder);
   endif
 
   fid = fopen (opts.name, "w+");
   if (fid == -1)
     error ("print: cannot open file %s for writing", opts.name);
   endif
   fclose (fid);
   unlink (opts.name);
-  
+
   opts.pstoedit_cmd = @pstoedit;
   opts.fig2dev_cmd = @fig2dev;
   opts.latex_standalone = @latex_standalone;
   opts.lpr_cmd = @lpr;
   opts.epstool_cmd = @epstool;
 
   if (isempty (opts.figure) || ! isfigure (opts.figure))
     error ("print: no figure to print");
diff --git a/scripts/plot/util/private/__print_parse_opts__.m b/scripts/plot/util/private/__print_parse_opts__.m
--- a/scripts/plot/util/private/__print_parse_opts__.m
+++ b/scripts/plot/util/private/__print_parse_opts__.m
@@ -257,17 +257,17 @@ function arg_st = __print_parse_opts__ (
   if (strcmp (__graphics_toolkit__, "gnuplot"))
     suffixes(strncmp (dev_list, "epslatex", 8)) = {"eps"};
   endif
 
   match = strcmpi (dev_list, arg_st.devopt);
   if (any (match))
     default_suffix = suffixes{match};
   else
-    error ("print: unknown device %s", arg_st.devopt);  
+    error ("print: unknown device %s", arg_st.devopt);
   endif
 
   if (dot == 0 && ! isempty (arg_st.name))
     arg_st.name = [arg_st.name "." default_suffix];
   endif
 
   if (arg_st.append_to_file)
     if (isempty (arg_st.name))
